planetary-social/nos
# Re-enable Xcode 15.2 support

## Issues covered
#1117 

## Description
This re-enables Xcode 15.2 support (thanks to Matt), a version of Xcode that's able to successfully build for previews after a clean.

## How to test
1. Open Xcode 15.2
2. Clean
3. Open AboutView or any view with previews.
4. Build previews and observe

Expected: Preview appears.

You could also try this with Xcode 15.3, though you probably have to build for testing there before you can see previews. Previews there work for me after building for testing.

Here's how it all works for me on this branch:

| Xcode version | Clean, Build Previews | Clean, Build for Testing, Build Previews |
|--------|--------|--------|
| 15.2 | ✅ | ✅ |
| 15.3 | ❌ | ✅ |
| 15.4 | ❌ | ❌ (does not build) |
| 16.0b1 | ❌ | ❌ (does not build) | 
| 16.0b2 🆕 | ❌ | ❌ (does not build) |
